instrumentation/netty/netty-4.1/library/src/main/java/io/opentelemetry/instrumentation/netty/v4_1/NettyClientTelemetryBuilder.java

```diff
@@ -24,17 +25,22 @@
 public final class NettyClientTelemetryBuilder {
 
   private final DefaultHttpClientInstrumenterBuilder<HttpRequestAndChannel, HttpResponse> builder;
-  private boolean emitExperimentalHttpClientEvents = false;
+  private boolean emitExperimentalTelemetry = false;
 
   NettyClientTelemetryBuilder(OpenTelemetry openTelemetry) {
     builder =
         NettyClientInstrumenterBuilderFactory.create("io.opentelemetry.netty-4.1", openTelemetry);
   }
 
+  /**
+   * @deprecated Use {@link Experimental#setEmitExperimentalTelemetry(NettyClientTelemetryBuilder,
+   *     boolean)} instead.
+   */
+  @Deprecated
   @CanIgnoreReturnValue
   public NettyClientTelemetryBuilder setEmitExperimentalHttpClientEvents(
-      boolean emitExperimentalHttpClientEvents) {
-    this.emitExperimentalHttpClientEvents = emitExperimentalHttpClientEvents;
+      boolean emitExperimentalTelemetry) {
+    this.emitExperimentalTelemetry = emitExperimentalTelemetry;
     return this;
   }
```

Comment:
I combined this with emitExperimentalTelemetry, which matches how it's enabled under single java agent flag `otel.instrumentation.http.client.emit-experimental-telemetry`